data.sql

```diff
@@ -0,0 +1,131 @@
+/* Populate database with sample data. */
+
+INSERT INTO animals (name, date_of_birth, escape_attempts,neutered, weight_kg)
+VALUES ('Agumon', '2020-02-03', 0, '1', 10.23);
+
+INSERT INTO animals (name, date_of_birth, escape_attempts,neutered, weight_kg)
+VALUES ('Gabumon', '2018-11-15', 2, '1', 8);
+
+INSERT INTO animals (name, date_of_birth, escape_attempts,neutered, weight_kg)
+VALUES ('Pikachu', '2021-01-07', 1, '0', 15.04);
+
+INSERT INTO animals (name, date_of_birth, escape_attempts,neutered, weight_kg)
+VALUES ('Devimon', '2017-05-12', 5, '1', 11);
+
+INSERT INTO animals (name, date_of_birth, escape_attempts,neutered, weight_kg)
+VALUES ('Charmander', '2020-02-08', 0, '0', -11);
+
+INSERT INTO animals (name, date_of_birth, escape_attempts,neutered, weight_kg)
+VALUES ('Plantmon', '2021-11-15', 2, '1', -5.7);
+
+INSERT INTO animals (name, date_of_birth, escape_attempts,neutered, weight_kg)
+VALUES ('Squirtle', '1993-04-02', 3, '0', -12.13);
+
+INSERT INTO animals (name, date_of_birth, escape_attempts,neutered, weight_kg)
+VALUES ('Angemon', '2005-06-12', 1, '1', -45);
+
+INSERT INTO animals (name, date_of_birth, escape_attempts,neutered, weight_kg)
+VALUES ('Boarmon', '2005-06-07', 7, '1', 20.4);
+
+INSERT INTO animals (name, date_of_birth, escape_attempts,neutered, weight_kg)
+VALUES ('Blossom', '1998-10-13', 3, '1', 17);
+
+INSERT INTO animals (name, date_of_birth, escape_attempts,neutered, weight_kg)
+VALUES ('Ditto', '2022-05-14', 4, '1', 22);
```

Comment:
-  [ ] It would be nice if you replace 0 with false and 1 with true in all your insertions as suggested by the previous reviewer 👍 Yeah I can understand that you are using 0 for false or 1 for true but it would be great if you directly use true and false in all the insertions 😊 like below ⬇️

```SQL
INSERT INTO animals (name, date_of_birth, escape_attempts,neutered, weight_kg)
VALUES ('Gabumon', '2018-11-15', 2, 'true', 8);

INSERT INTO animals (name, date_of_birth, escape_attempts,neutered, weight_kg)
VALUES ('Pikachu', '2021-01-07', 1, 'false', 15.04);
```